src/app/components/hris/employees/employee-profile/accordions/accordion-profile/accordion-career-additional-information/accordion-career-additional-information.component.ts

```diff
@@ -0,0 +1,251 @@
+import { EmployeeDataService } from 'src/app/services/hris/employee/employee-data.service';
+import { EmployeeData } from 'src/app/models/hris/employee-data.interface';
+import { ClientService } from 'src/app/services/hris/client.service';
+import { EmployeeTypeService } from 'src/app/services/hris/employee/employee-type.service';
+import { CustomFieldService } from 'src/app/services/hris/field-code.service';
+import { CustomField } from 'src/app/models/hris/custom-field.interface';
+import { SimpleEmployee } from 'src/app/models/hris/simple-employee-profile.interface';
+import { AuthAccessService } from 'src/app/services/shared-services/auth-access/auth-access.service';
+import { SharedPropertyAccessService } from 'src/app/services/hris/shared-property-access.service';
+import { PropertyAccessLevel } from 'src/app/models/hris/constants/enums/property-access-levels.enum';
+import { SharedAccordionFunctionality } from 'src/app/components/hris/employees/employee-profile/shared-accordion-functionality';
+import { EventEmitter } from '@angular/core';
+import { NavService } from 'src/app/services/shared-services/nav-service/nav.service';
+
+@Component({
+  selector: 'app-accordion-career-additional-information',
+  templateUrl: './accordion-career-additional-information.component.html',
+  styleUrls: [ './accordion-career-additional-information.component.css' ]
+})
+export class AccordionCareerAdditionalInformationComponent {
+  screenWidth = window.innerWidth;
+
+  @HostListener('window:resize', [ '$event' ])
+  usingProfile: boolean = true;
+
+  onResize() {
+    this.screenWidth = window.innerWidth;
+  }
+
+  @Output() updateEmployeeProfile = new EventEmitter<any>();
+  @Input() employeeProfile!: { employeeDetails: EmployeeProfile, simpleEmployee: SimpleEmployee }
+
+  customFields: CustomField[] = [];
+  additionalFormProgress: number = 0;
+  employeeId: number | undefined;
+  loggedInProfile!: EmployeeData;
+
+  constructor(
+    private fb: FormBuilder,
+    private employeeService: EmployeeService,
+    private snackBarService: SnackbarService,
+    private employeeProfileService: EmployeeProfileService,
+    private employeeDataService: EmployeeDataService,
+    private clientService: ClientService,
+    private employeeTypeService: EmployeeTypeService,
+    private customFieldService: CustomFieldService,
+    public authAccessService: AuthAccessService,
+    public sharedPropertyAccessService: SharedPropertyAccessService,
+    public sharedAccordionFunctionality: SharedAccordionFunctionality,
+    public navService: NavService) {
+  }
+
+  ngOnInit() {
+    this.usingProfile = this.employeeProfile!.simpleEmployee == undefined;
+    this.loggedInProfile = this.navService.getEmployeeProfile();
+    const id = this.employeeProfile.employeeDetails.id ? this.employeeProfile.employeeDetails.id : this.employeeProfile.simpleEmployee.id;
+    this.employeeId = id;
```

Comment:
cant you just assign it directly to this.employeeId ?
if possible it might be better on line 62 and 63?